app/js/actions/plugins.js

```diff
@@ -20,7 +64,8 @@ export const loadWorkflows = () => {
             .then((response) => (response.json()))
             .then((json) => {
                 Object.keys(json.workflows).map(workflow =>
-                    dispatch(foundWorkflow(plugin.name, json.workflows[workflow]))
+                    dispatch(foundWorkflow(plugin.name, json.workflows[workflow])) &&
```

Comment:
The `&&` is kind of a hack, this is only working because `dispatch` is returning a promise or something right? Which is truthy enough to cause JavaScript to move on to the next expression.

Since we don't do anything with the result of map, let's change it to 
```JavaScript
Object.keys(json.workflows).forEach(workflow => {
   dispatch(...);
   dispatch(...);
})
```